lib/semver_checker.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+var semver = require("semver");
+var pkg = require("../package.json");
+var discover = require("./util/discover");
+
+function autoDiscoverModules(root) {
+  return discover.all(root).modules;
+}
+
+module.exports = function(eyeglass, sass, options) {
+  var root = options.root;
+  var strictMode = options.strictModuleVersions;
+  var modules = autoDiscoverModules(root);
+
+  var incompatible = {};
+  var noEngine = [];
+  var hasErrors = false;
+  var hasWarnings = false;
+
+  strictMode = (typeof strictMode === "undefined") ? "warn" : strictMode;
+
+  modules.forEach(function(m) {
+    if (!m.eyeglass || !m.eyeglass.needs) {
```

Comment:
In the pull request description it was `eyeglass.engine` but all the code seems to be using `needs` which I like better.